src/Aquarius.ONE.ClientSDK/Common/Activity/ActivityApi.cs

```diff
@@ -99,14 +99,14 @@ public ActivityApi(PlatformEnvironment environment, bool continueOnCapturedConte
 
                 Event(null,
                     respContent.ResponseMessage.IsSuccessStatusCode
-                        ? CreateLoggerArgs(EnumLogLevel.Trace, "GetActivitiesAsync Success", respContent.ResponseMessage.StatusCode, watch.ElapsedMilliseconds)
-                        : CreateLoggerArgs(EnumLogLevel.Warn, "GetActivitiesAsync Failed", respContent.ResponseMessage.StatusCode, watch.ElapsedMilliseconds));
+                        ? CreateLoggerArgs(Proto.EnumOneLogLevel.OneLogLevelTrace, "GetActivitiesAsync Success", respContent.ResponseMessage.StatusCode, watch.ElapsedMilliseconds)

```

Comment:
In this class you use the "Proto." object reference and in the other files below you don't. Is there a particular reason?

Reply:
It was not a conscious decision, I put in EnumOneLogLevel and told it to import the required using, I'm guessing this file already had the Proto alias while others did not.  It's possible that something in this file had a previous conflict resulting in the use of the alias.

Reply:
My goal here was to get rid of the obsolete warnings, there were nearly 700 of them, without getting bogged down in attempting to clean up the code.  Strictly speaking this story only required the addition of the new subTypes and configType which was accomplished by updating the Models.CSharp package.  The removal of EnumLogLevel was a bonus.